COPYING-BSD

```diff
@@ -0,0 +1,27 @@
+Copyright (C) 2012-2018, Blue Brain Project/EPFL.
```

Comment:
I assume there were changes in the software since 2018? The last date should reflect the year of the last changes. Thx

Reply:
ok